cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -808,4 +810,23 @@ public String toString() {
             });
     }
 
+    public class PodRef {
+
+        private String podName;
+        private int podIndex;
+
+        public PodRef(String podName, int podIndex){
+            this.podName = podName;
+            this.podIndex = podIndex;
+        }
+
+        public String getPodName() {
+            return podName;
+        }
+
+        public int getPodIndex() {
+            return podIndex;
+        }
+
```

Comment:
You can simplify all you logging calls by overriding `toString`. You should probably also override `equals` and `hashCode` too, in terms of the `podName`.

Reply:
Should I create the `hashcode` and `equals` method ? They are not being used in the current code

Reply:
I would, personally.